### PR TITLE
NO-ISSUE: MicroShift cluster-bot: install make

### PR DESCRIPTION
**Pull request by pmtk**
Needed because of https://github.com/openshift/microshift/pull/2760

### CODE REVIEW

File: ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh [hunk 1]
```diff
@@ -42,8 +42,12 @@ if ! sudo subscription-manager status >&/dev/null; then
...
 fi
 
-sudo dnf clean all
-hash git || sudo dnf install -y git-core
+PKGS="git-core make"
+sudo dnf clean all --enablerepo=\*
+if ! sudo dnf install -y ${PKGS}; then
+	sudo rm -rf /var/cache/dnf
+	sudo dnf install -y ${PKGS}
+fi
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh:50]: I would try something similar to dnf_retry here: 
https://github.com/openshift/microshift/blob/main/scripts/devenv-builder/configure-vm.sh#L30

Deleting the directory should be no different than clean

**Comment by pmtk** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-clusterbot

**Pull request closed without merging**